src/components/common/Extensions/ExtensionUpgradeButton/ExtensionUpgradeButton.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useCallback } from 'react';
+import {
+  Extension,
+  ExtensionVersion,
+  isExtensionCompatible,
+} from '@colony/colony-js';
+
+import { ActionButton } from '~shared/Button';
+import { InstalledExtensionData } from '~types';
+import { ActionTypes } from '~redux/index';
+import { mapPayload } from '~utils/actions';
+import { useAppContext, useColonyContext } from '~hooks';
+
+interface Props {
+  extensionData: InstalledExtensionData;
+}
+
+const ExtensionUpgradeButton = ({ extensionData }: Props) => {
+  const { colony } = useColonyContext();
+  const { user } = useAppContext();
+
+  const transform = useCallback(
+    mapPayload(() => ({
+      colonyAddress: colony?.colonyAddress,
+      extensionId: extensionData.extensionId,
+      version: extensionData.currentVersion + 1,
+    })),
+    [],
+  );
+
+  // const isSupportedColonyVersion =
+  //   parseInt(colonyVersion || '1', 10) >= ColonyVersion.LightweightSpaceship;
+  const isSupportedColonyVersion = true;
```

Comment:
This shouldn't be commented out anymore.

Basically you need to check the new versions format of ColonyJS and import them, then update the above check. 

_(basically we check if colonies are newer than version 5 of the contract, which is a legacy thing)_

Reply:
@rdig I think the problem here is that #102 isn't done yet so we don't have access to `colonyVersion`. Please correct me if I'm wrong and we can get it somewhere else, but otherwise I opened #153 to track it

Reply:
Oh. I had a vague idea that is was already implemented.

In that case we'll just wait for it to be done.

@Julianahlk leave it as it is for the time being